locations/fblipke.yml

```diff
@@ -0,0 +1,152 @@
+---
+location: fblipke
+location_nice: Marzahn - Blumberger Damm 305
+latitude: 52.557725
+longitude: 13.57437
+altitude: 42
+height: 13
+contact_name: 'Guido L.'
+contact_nickname: 'LipkeGu'
+contacts:
+  - 'contact@fblipke.de'
+  - '@lipkegu:matrix.org'
+hosts:
+# Proxmox VM (QEMU)
+  - hostname: fblipke-core
+    role: corerouter
+    model: "x86-64"
+    host__packages__to_merge:
+      - "nano qemu-ga kmod-brcmfmac kmod-brcmutil brcmfmac-firmware-4366c0-pcie"
+    wireless_profile: fblipke
+    wifi_roaming: true
+
+#  NSM5
+  - hostname: fblipke-ap01
+    role: ap
+    model: "ubnt_nanostation_m5_xw"
+    wireless_profile: fblipke
+    wifi_roaming: true
```

Comment:
Not sure if this is a good idea for a device with only 8 MB of flash and 64 MB of memory as it adds quite a bit to the image size. Personally I would not use roaming for your setup. I would use the default wireless profile for the Nanostation and only use the custom wireless profile at the core.